README.md

```diff
@@ -1,10 +1,79 @@
-# ENV SSM
-Gets params from your environment first, then from the ssm parameter store.
+# env-ssm
+
+Load environment from SSM Parameter Store.
+
+Variables load (and are overwritten if duplicates are found) in the following order:
+
+1. Load `SSM`
+2. Load `.env` file
+3. Load `.tfvars` file
```

Comment:
Remind me why .tfvars file overwrites values in the .env file?

Reply:
It's because tfvars are turned off by default. If a developer specifies a tfvars, it's a bit more intentional than a .env file.